web/js/main.js

```diff
@@ -182,7 +184,9 @@ window.onload = () => {
         }
       });
     }
-
+    if (parameters.mockFutureLayer) {
```

Comment:
So I noticed that if you move this call to `mockFutureTimeLayerOptions` above the call to `adjustEndDates`...